### PR TITLE
Skip Failing Test Blocking CI

### PR DESCRIPTION
**Pull request by CodaFi**

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by neonichu** [comment]
We saw this in several different tests yesterday (e.g. `WorkspaceTests.testSimpleAPI`), so I suspect skipping a single test will not help much.

File: Tests/WorkspaceTests/InitTests.swift [hunk 1]
```diff
@@ -266,6 +266,8 @@ class InitTests: XCTestCase {
     }
     
     func testNonC99NameExecutablePackage() throws {
+        throw XCTSkip("This test fails to find XCTAssertEqual; rdar://101868275")
```
Comment (tomerd) [inline Tests/WorkspaceTests/InitTests.swift:269]: cc @neonichu

**Comment by neonichu** [comment]
This skips the other two failures we're currently seeing around 20% of the time: https://github.com/apple/swift-package-manager/pull/5868

**Comment by neonichu** [comment]
@swift-ci smoke test

**Pull request merged**